Assets/Shopify/UIToolkit/Test/Integration/TestRemoteImageLoaderIntegration.cs

```diff
@@ -0,0 +1,29 @@
+namespace Shopify.UIToolkit.Test.Integration {
+	using UnityEngine;
+    using UnityEngine.UI;
+    using UnityEngine.TestTools;
+    using System.Collections;
+    using Shopify.Unity.SDK;
+
+	[IntegrationTest.DynamicTest("UIToolkitIntegrationTests")]
+    [IntegrationTest.Timeout(10)]
+    public class TestRemoteImageLoaderIntegration {
+
+        [UnityTest] 
+        public IEnumerator TestDownloadImage() {
+            var gameObject = new GameObject();
+            var loader = gameObject.AddComponent<RemoteImageLoader>();
+
+            loader.ImageURL = "https://cdn.shopify.com/s/files/1/2094/7261/products/product-image-305453751.jpg?v=1497377691";
+            loader.LoadImage();
+
+            var image = gameObject.GetComponent<Image>();
+            while (image.sprite == null) {
```

Comment:
same deal, if the network is broken on ci or on someones machine this tests will run forever... you might need to check the error as well here or include an overload to loader.loadimage that notifies when the request is complete (I like it with no parameters too, but in testing and for some cases a more robust method might be better?